src/devtools/client/inspector/markup/components/MarkupApp.js

```diff
@@ -3,25 +3,28 @@ const dom = require("react-dom-factories");
 const PropTypes = require("prop-types");
 const { connect } = require("react-redux");
 
+const { getNode, getRootNodeId } = require("../reducers/markup");
+
 const Node = createFactory(require("./Node"));
 
 const Types = require("../types");
 
 class MarkupApp extends PureComponent {
   static get propTypes() {
     return {
-      markup: PropTypes.shape(Types.markup).isRequired,
+      node: PropTypes.shape(Types.node).isRequired,
       onSelectNode: PropTypes.func.isRequired,
       onShowEventTooltip: PropTypes.func.isRequired,
       onToggleNodeExpanded: PropTypes.func.isRequired,
+      onMouseEnterNode: PropTypes.func.isRequired,
+      onMouseLeaveNode: PropTypes.func.isRequired,
```

Comment:
Minor comment, but i wonder if we want to continue using proptypes? 

I don't think it gets us much when we're using a type system.

Reply:
Yeah, I usually leave them in to check calls from javascript files. But when all callers are already converted to Typescript, the proptypes probably don't add any value anymore. Furthermore, I noticed that they can actually get in the way: Typescript somehow tries to derive type information about the component from proptypes, but the result didn't work for me and I had to remove the proptypes for Typescript to respect my manual typings.